test/integration/features/basic.feature

```diff
@@ -3,14 +3,16 @@ Feature: Basic
   As a user
   I need to be able to bring up a test environment
 
-  Scenario: Basic test scenario
-    Given executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+  Scenario: Starting Minishift
+    Given Minishift should have state "Does Not Exist"
```

Comment:
@hferentschik One another thing is that in future we will probably have more scenarios on minishift start. For example stopped -> running, running -> running. These are different test cases, so "minishift has state" is needed.